File: src/main/java/uk/org/scbc/entities/Item.java
package uk.org.scbc.entities;

import lombok.Data;

import javax.persistence.*;

/**
 * Created by marc.thomas on 25/04/2016.
 */
@Data
@Entity
@Table(name = "items")
public class Item {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Integer id;

    private String name;
    private String description;
    private String notes;
    private Float value;
    private Boolean isAsset;

    @ManyToOne
    private Storage storage;

    @ManyToOne
    private Organisation organisation;

    private String reference;
}

File: src/main/resources/static/Item.js
const React = require('react');

export default class Item extends React.Component {
    render() {
        return (
            <tr>
                <td>{this.props.item.name}</td>
                <td>{this.props.item.description}</td>
                <td>{this.props.item.notes}</td>
                <td>{this.props.item.value}</td>
                <td>{this.props.item.isAsset}</td>
                <td>{this.props.item.reference}</td>
                <td>{this.props.item.storage}</td>
                <td>{this.props.item.organisation}</td>
            </tr>
        )
    }
}